src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -116,7 +116,7 @@
         </div>
 
         <!-- Versioning -->
-        <fieldset *ngIf="editing">
+        <fieldset *ngIf="editing && !bucketForm.getValue('lock_enabled')">
```

Comment:
Just a question: would it be more clear for the user if we show the versioning state but in read-only mode (not able to uncheck) so he clearly understands that object locking implies enabling versioning ?

Reply:
Yes makes sense. I've addressed your suggestion, thanks!